src/CONST.ts

```diff
@@ -5496,6 +5499,30 @@ const CONST = {
             REMOVE: 'remove',
         },
     },
+
+    CSV_IMPORT_COLUMNS: {
```

Comment:
I get the idea, but it is not needed IMO and in function `findColumnName` we don't know it e.g. field `Name` is from `categories` or `tags`, so what should be set there as an attribute? (there is no difference, because they are the same) Example below:

>         case 'glcode':
>        case 'gl':
>            attribute = CONST.CSV_IMPORT_COLUMNS.CATEGORIES.GL_CODE;
>            break;